package.json

```diff
@@ -63,8 +63,9 @@
 		"start": "meteor npm i && meteor",
 		"debug": "meteor run --inspect",
 		"debug-brk": "meteor run --inspect-brk",
-		"lint": "eslint .",
-		"lint-fix": "eslint . --fix",
```

Comment:
I've removed the fix job because it can be done like this: `meteor npm run core-lint -- --fix` or `meteor npm run livechat-lint -- --fix`